pkg/capabilities/capabilities.go

```diff
@@ -91,18 +91,21 @@ func ValidateCapabilities(caps []string) error {
 // MergeCapabilities computes a set of capabilities by adding capapbitilities
 // to or dropping them from base.
 //
-// Note that "ALL" will cause all known capabilities to be added/dropped but
+// Note that "ALL" in capAdd will cause all known capabilities to be added
+// "All" in capDrop will cause all capabilities to be dropped except those that are added
 // the ones specified to be dropped/added.
```

Comment:
This last comment line (line 96) no longer makes sense.

Reply:
What I am trying to say is --cap-drop=all --cap-add=sys_admin, you end up with sys_admin.

Reply:
OK, then I think you can just remove line 96 and you'll be good.